examples/fair_prep_results.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import os
+os.chdir("..")
+import shutil
+import warnings
+warnings.simplefilter('ignore')
+
+from fp.traindata_samplers import CompleteData
+from fp.missingvalue_handlers import CompleteCaseAnalysis
+from fp.dataset_experiments import GermanCreditDatasetSexExperiment
+from fp.dataset_experiments import RicciRaceExperiment
```

Comment:
Why is this imported?

Reply:
It was done for working with RicciDataset in this file, will do it separately, has been removed for here.